NitroxServer/ConsoleCommands/ChangeServerGamemodeCommand.cs

```diff
@@ -18,9 +18,23 @@ public ChangeServerGamemodeCommand(ServerConfig serverConfig) : base("changeserv
         protected override void Execute(CallArgs args)
         {
             ServerGameMode sgm = args.Get<ServerGameMode>(0);
+            bool currentGamemodeHardcore = serverConfig.IsGamemodeHardcore();
 
             serverConfig.GameModeEnum = sgm;
             SendMessage(args.Sender, $"Server gamemode changed to \"{sgm}\" by {args.SenderName}");
+            
+            if (serverConfig.IsGamemodeHardcore())
+            {
+                serverConfig.DisableAutoSave = true;
```

Comment:
Why are we disabling auto save for hardcore?

Reply:
I wasn't sure if to do this or not. In the base game using the save menu isn't allowed just the save and quit. Happy to remove

Reply:
As far as I remember vanilla SN "saves" when you enter a base and that isn't handled by us.

Reply:
Taken from the subnautica game modes "In addition, you cannot save the game anytime like in the other modes; the only option is to Save & Quit" so I assume that in hardmode mode it doesn't save inside the base

Reply:
I think there will always be many ways to circumvent this.  It doesn't look like we are suppressing the save command.  Also, I think this would more likely cause an issue if something crashed.  Would prefer to keep it and just blacklist the dead players.